src/main/java/com/technicjelle/bluemapofflineplayermarkers/Main.java

```diff
@@ -22,6 +26,16 @@ public final class Main extends JavaPlugin implements Listener {
 	public void onEnable() {
 		Metrics metrics = new Metrics(this, 16425);
 
+		getDataFolder().mkdirs();
+		File configFile = new File(getDataFolder(), "config.yml");
+		if (!configFile.exists()) {
+			try (InputStream inputStream = Main.class.getClassLoader().getResourceAsStream("config.yml")) {
+				Files.copy(inputStream, configFile.toPath());
```

Comment:
I [recently](https://github.com/TechnicJelle/BlueMapChatMarkers/blob/main/src/main/java/com/technicjelle/bluemapchatmarkers/Main.java#L49-L56) did this like so:
```java
try {
	Files.copy(getResource("textStyle.css"), configFile);
```
Feels a lot simpler to me, so without those classloaders and streams :)